assets/shaders/naturalA.fsh

```diff
@@ -63,16 +63,12 @@ void main()
     float d2=dot(abs(c20-c02),dt)+0.0001;
     float hl=dot(abs(c01-c21),dt)+0.0001;
     float vl=dot(abs(c10-c12),dt)+0.0001;
-
-    float md = d1+d2;   float mc = hl+vl;
-    hl*=  md;vl*= md;   d1*=  mc;d2*= mc;

```

Comment:
So you just remove these multiplications, won't that change the look? No idea what's going in this shader anyway though :)

I'm guessing what happened before sometimes is that md and mc both became 0, ww became 0, and then the long line below divides by (3.0*ww) which could end up 0.0. And dividing by 0.0 is bad... Maybe better to just check for that? Not sure.

Reply:
Nope. It looks the same (99% +)

That was the only solution that came to my mind. Don't merge for now to see if I can find a better fix.

Reply:
@hrydgard 
Quick question: If this was because of a miscalculation, then wouldn't the issue occur on all backends? But this only happens on OpenGl ES (not even OpenGl).